dnsmasq-metrics/pkg/test/test.go

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
I found this file name is somehow confusing --- I thought there might be tests in it, but turned out it is like test utils...